modules/NetworkLogger.js

```diff
@@ -33,6 +33,9 @@ export default {
               if (Platform.OS === 'android') {
                 Instabug.networkLog(JSON.stringify(network));
               } else {
+                if (typeof network.requestBody !== "string") {
+                  network.requestBody = JSON.stringify(network.requestBody);
```

Comment:
It's best to keep the data manipulation in one place. Could you move the formatting to the Xhr 
interceptor please?

Reply:
Yes, sure.